src/sqlancer/Randomly.java

```diff
@@ -452,6 +451,13 @@ public long getLong(long left, long right) {
         return getNextLong(left, right);
     }
 
+    public BigInteger getBigInteger(BigInteger left, BigInteger right) {
+        if (left.equals(right)) {
+            return left;
+        }
+        return new BigInteger(String.valueOf(getInteger(left.intValue(),right.intValue())));
```

Comment:
I think this might cause some issues. According to https://docs.oracle.com/javase/8/docs/api/java/math/BigInteger.html#intValue, this method truncates the integer, so it might return a number outside the range specified by the big integers. Perhaps we can first check whether the big integer can be precisely represented, and if not, throw an `IgnoreMeException`?

Reply:
OK, IgnoreMeException  added